Chiazokam/Politico
# #163356516 create the get one office endpoint

#### What does this PR do?
This PR tests and creates the GET one office endpoint

#### Description of Task to be completed?
- write tests for all the functions
- create the endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-datastructures-get-one-office-163356516`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Make a `GET` request to `http://localhost:3000/api/v1/offices/1 `
- You should receive a successful response

#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[#163356516](https://www.pivotaltracker.com/story/show/163356516)

####  Images
![image](https://user-images.githubusercontent.com/26940294/51780543-ab14bd80-210f-11e9-82a6-b77396faec39.png)
